app/decorators/service_provider_session_decorator.rb

```diff
@@ -111,11 +107,11 @@ def sp_alert?
   end
 
   def sp_alert_name
-    sp_alert? ? SP_ALERTS[sp_name][:i18n_name] : nil
+    SP_ALERTS.dig(sp_name, :i18n_name) || nil
```

Comment:
If `SP_ALERTS[sp_name][:i18n_name]` doesn't exist, it'll return `nil` anyway, so `|| nil` isn't really necessary.